### DISCUSSION TITLE
How can evaluate Conditional attributes in for loop ?

### DISCUSSION THREAD

**Discussion by rgunindi** [a-h/templ#577] [Q&A] [answered]
```	
for _, account := range accounts {
	<div class="account-info">
	<label for="accountId">Account ID</label>
	<input type="text" id="accountId" name="accountId" value={ account.ID } 
	 if true {
      class="itIsTrue"
             }
            }
        />
      </div>
 }
```

I can't use the code at the top, is there any way to do this?

**Comment by rgunindi** [comment]
any one?

**Reply by a-h** [reply]
You can create a map of class names to bool values. If the bool is true, the class is applied.

It's covered in the docs at:

https://templ.guide/syntax-and-usage/css-style-management#dynamic-class-names

**Comment by rgunindi** [comment]
Can we pass attributes directly to HTML elements? ex disabled .

This is not working
`<input type="text" id="username" value={account.name} templ.KV("disabled", account.IsAccountActive)/>`

**Reply by a-h** [reply, marked as answer]
See https://templ.guide/syntax-and-usage/attributes#boolean-attributes in the docs:

```
<input type="text" id="username" value={account.name} disabled?={ !account.IsAccountActive }>{ text }</button>
```
